.gitignore

```diff
@@ -58,3 +58,5 @@ junit.*.xml
 
 # test results
 _artifacts
+awsiamconfiguration.yaml
```

Comment:
do we need these? thought they also get created under the artifacts folder

Reply:
Haven't got to the bottom as to why, but sometimes it gets created in the test directory. Will open a follow up issue to resolve than try to solve it now.

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/1827